test/cri-containerd/container_test.go

```diff
@@ -131,3 +137,81 @@ func Test_RotateLogs_LCOW(t *testing.T) {
 		expected++
 	}
 }
+
+func Test_RunContainer_Events_LCOW(t *testing.T) {
+	pullRequiredLcowImages(t, []string{imageLcowK8sPause, imageLcowAlpine})
+	client := newTestRuntimeClient(t)
+
+	podctx, podcancel := context.WithCancel(context.Background())
+	defer podcancel()
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
I dont understand this cancel or why its used. The event service appears to never be canceled anyways?

Reply:
iirc, this was just to create a timeout in the event that the test hung forever waiting for the proper events or an error from the channels.

Reply:
Oh lol. It has a timeout! It cancels itself :)